dts/bindings/gpio/nxp,pca9538.yaml

```diff
@@ -0,0 +1,5 @@
+# Copyright 2024 NXP
+# SPDX-License-Identifier: Apache-2.0
+description: NXP PCA9538 I2C GPIO expander node
+compatible: nxp,pca9538
+include: nxp,pca_series.yaml
```

Comment:
maybe I should add "ngpios" const property here, instead of hard-coding it in driver (macro `GPIO_PCA_GET_PORT_NO_BY_PART_NO`)? since now we are creating separate bindings for each device.
for example:

```yaml
properties:
  ngpios:
    required: true
    const: 8
```

Note that port_no is also hard-coded into each `struct gpio_pca_series_part_config` instance. I probably should remove it there, since maintaining port_no or pin_count at multiple places is inconvinient and error-prone.

Reply:
optimized.